### PR TITLE
Kin 8 movies crud pull request ;) <3 

### PR DESCRIPTION
**Pull request by Hjordten**
Somebody please accept my pull request ;) <3

### CODE REVIEW

File: src/main/java/com/example/kinoxpback/model/Movie.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
+package com.example.kinoxpback.model;
+
+//import com.example.kinoxpback.model.Showing;
+import com.example.kinoxpback.enums.AgeLimit;
+import com.example.kinoxpback.enums.MovieCategory;
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import jakarta.persistence.*;
+
```
Comment (KEATLLoevkilde) [inline src/main/java/com/example/kinoxpback/model/Movie.java:8]: hej

File: src/main/java/com/example/kinoxpback/model/Movie.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    @Column(name = "category")
+    private MovieCategory category;
+    @Enumerated(EnumType.STRING)
+    @Column(name = "age_limit")
+    private AgeLimit ageLimit;
+    @Column(name = "description", columnDefinition = "TEXT")
+    private String description;
+    @Column(name = "href")
+    private String href;
+
```
Comment (ghost) [inline src/main/java/com/example/kinoxpback/model/Movie.java:28]: Fjern udkommentering

File: src/main/java/com/example/kinoxpback/model/Movie.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    private String title;
+    @Enumerated(EnumType.STRING)
+    @Column(name = "category")
+    private MovieCategory category;
+    @Enumerated(EnumType.STRING)
+    @Column(name = "age_limit")
+    private AgeLimit ageLimit;
+    @Column(name = "description", columnDefinition = "TEXT")
+    private String description;
+    @Column(name = "href")
```
Comment (KEATLLoevkilde) [inline src/main/java/com/example/kinoxpback/model/Movie.java:26]: private String photo;

Comment (ghost) [inline src/main/java/com/example/kinoxpback/model/Movie.java]: @Column(name = "href", length = Integer.MAX_VALUE)
    private String photo;

File: src/main/java/com/example/kinoxpback/model/Movie.java [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+/*
+    @OneToMany(mappedBy = "movie", cascade = CascadeType.ALL)
+    @JsonBackReference
+    private Set<Showing> showings;
+ */
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
```
Comment (ghost) [inline src/main/java/com/example/kinoxpback/model/Movie.java:37]: Fix href naming

File: src/test/java/com/example/kinoxpback/MovieTest.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+import org.junit.jupiter.api.TestInstance;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class MovieTest {
+
+        MovieServiceImpl sut;
```
Comment (ghost) [inline src/test/java/com/example/kinoxpback/MovieTest.java]: Fix test

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/kinoxpback/model/Movie.java (hunk 3 above)
Comment (JesperZamora) [reply src/main/java/com/example/kinoxpback/model/Movie.java]: Done

File: src/main/java/com/example/kinoxpback/model/Movie.java (hunk 4 above)
Comment (JesperZamora) [reply src/main/java/com/example/kinoxpback/model/Movie.java:37]: Done

File: src/test/java/com/example/kinoxpback/MovieTest.java (hunk 5 above)
Comment (JesperZamora) [reply src/test/java/com/example/kinoxpback/MovieTest.java]: Done

**Review by ghost — APPROVED** [review]
Daddy is pleased

**Pull request merged**